pkg/schema/schema.json

```diff
@@ -19,16 +19,19 @@
                         [
                             {
                                 "type": "IP",
-                                "replacementType": "Consistent"
+                                "replacementType": "Consistent",
+                                "target": "All"
                             },
                             {
                                 "type": "MAC",
-                                "replacementType": "Random"
+                                "replacementType": "Random",
+                                "target": "FileContents"
                             }
                         ]
                     ],
                     "title": "Obfuscation Schema",
                     "type": "array",
+                    "minItems": 1,
```

Comment:
do you mind adding a small test file in pkg/schema/testfiles/malformed? to ensure that works? I think with that you can also skip the `if len(obfuscators) < 1 {` as the validation during the config read would catch it

Reply:
Validation for `minItems` has not been implemented in the [library](https://github.com/atombender/go-jsonschema#validation)

Reply:
crap. I'll add it over there, let's leave it here then.

Reply:
added it here: https://github.com/atombender/go-jsonschema/pull/30